crates/data_models/src/payments/payment_attempt.rs

```diff
@@ -107,6 +107,7 @@ pub struct PaymentAttempt {
     pub attempt_id: String,
     pub status: storage_enums::AttemptStatus,
     pub amount: i64,
+    pub net_amount: i64,
```

Comment:
How we are making this as mandatory field here?

Reply:
`net_amount` field in domain Attempt will always be calculated during conversion from diesel model to domain model.